composer.json

```diff
@@ -30,34 +28,61 @@
         "symfony/http-foundation": "^5.0",
         "symfony/event-dispatcher": "^5.0",
         "guzzlehttp/guzzle": "^6.3.0",
-        "ezsystems/ezplatform-kernel": "^1.2",
-        "ezsystems/ezplatform-richtext": "^2.0",
-        "ezsystems/ezplatform-admin-ui": "^2.2"
+        "ibexa/core": "^4.0@dev",
+        "ibexa/fieldtype-richtext": "^4.0@dev",
+        "ibexa/admin-ui": "^4.0@dev"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^3.2",
-        "phpmd/phpmd": "^2.6",
-        "friendsofphp/php-cs-fixer": "^2.10",
-        "phpunit/phpunit": "^6.4",
-        "ezsystems/ezplatform-code-style": "^0.1.0",
-        "dg/bypass-finals": "^1.3"
+        "dg/bypass-finals": "^1.3",
+        "phpunit/phpunit": "^9.5",
+        "ibexa/code-style": "^1.0",
+        "friendsofphp/php-cs-fixer": "^3.0",
+        "phpstan/phpstan": "^1.2",
+        "phpstan/phpstan-phpunit": "^1.0",
+        "phpstan/phpstan-symfony": "^1.0"
     },
     "autoload": {
         "psr-4": {
             "EzSystems\\EzPlatformAutomatedTranslation\\": "lib",
             "EzSystems\\EzPlatformAutomatedTranslationBundle\\": "bundle",
-            "EzSystems\\EzPlatformAutomatedTranslationBundle\\Tests\\": "tests"
+            "Ibexa\\AutomatedTranslation\\": "src/lib/",
+            "Ibexa\\Bundle\\AutomatedTranslation\\": "src/bundle/",
+            "Ibexa\\Contracts\\AutomatedTranslation\\": "src/contracts/"
```

Comment:
Those should go on top, given it's the physical namespace used in the files, while EzSystems is just an alias